src/main/java/com/salesforce/storm/spout/sideline/persistence/ZookeeperPersistenceAdapter.java

```diff
@@ -242,7 +242,7 @@ public void clearSidelineRequest(final SidelineRequestIdentifier id, final Consu
 
                 for (final String partition : partitions) {
                     consumerPartitions.add(
-                        new ConsumerPartition(namespace, Integer.valueOf(partition))
```

Comment:
Logically the same, Integer.valueOf() returns an Integer where Integer.parseInt() returns an int, which is what ConsumerPartition's constructor takes.